benchmarks/Schelling/schelling.py

```diff
@@ -31,7 +42,7 @@ def step(self):
 
         # If unhappy, move:
         if similar < self.homophily:
-            self.move_to(self.model.grid.select_random_empty_cell())
+            self.cell = self.model.grid.select_random_empty_cell()
```

Comment:
If we have both, then why is this not `self.move_to`?

Reply:
please read the discussion both here and in #2292.

Reply:
I am sorry for the short answer and the merge, but move_to is added back in via #2292.

Reply:
I see, then you have my post-merge approval.